packages/react/src/components/FormHelperText/FormHelperText.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import {ArgsTable, Source, Story, Canvas, Meta} from '@storybook/addon-docs';
+import dedent from 'ts-dedent';
+import StoryConfig from '../../../.storybook/story-config.ts';
+import FormHelperText from './FormHelperText.tsx';
+
+export const meta = {
+  component: FormHelperText,
+  title: StoryConfig.FormHelperText.hierarchy,
+};
+
+<Meta title={meta.title} component={meta.component} />
+
+export const Template = args => <FormHelperText {...args} />
+
+# Form Helper Text
+
+- [Overview](#overview)
+- [Props](#props)
+- [Usage](#usage)
+
+## Overview
+
+Use `FormHelperText` to get data from the user.
+
+<Canvas>
+  <Story name="Overview" args={{children: 'Helper Text'}}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+## Props
+
+<ArgsTable story="Overview" />
+
+## Usage
+
+Import and use the `FormHelperText` component in your components as follows.
+
+<Source
+  language="jsx"
+  dark
+  format
+  code={dedent`import FormHelperText from '@oxygen-ui/react/FormHelperText';\n`}
```

Comment:
Shall we also show the implementation?

Reply:
Fixed with https://github.com/wso2/oxygen-ui/pull/38/commits/f2ee9ffa587d3b161c78343d67540bd081438984